src/libs/OptionsListUtils.js

```diff
@@ -191,8 +203,13 @@ function getSearchText(report, reportName, personalDetailList, isChatRoomOrPolic
 
         if (isChatRoomOrPolicyExpenseChat) {
             const chatRoomSubtitle = ReportUtils.getChatRoomSubtitle(report, policies);
-            Array.prototype.push.apply(searchTerms, chatRoomSubtitle.split(''));
-            Array.prototype.push.apply(searchTerms, chatRoomSubtitle.split(','));
+
+            // When running tests, chatRoomSubtitle can be undefined due to the Localize() stuff being mocked in the tests.
+            // It's OK to ignore this and just add a null check in here to keep code from crashing.
+            if (chatRoomSubtitle) {
+                Array.prototype.push.apply(searchTerms, chatRoomSubtitle.split(''));
+                Array.prototype.push.apply(searchTerms, chatRoomSubtitle.split(','));
```

Comment:
We can use a regex here, right? Something like `.split(/[o\s]/)`

Reply:
Thoughts on this, @tgolen?

Reply:
Oh, sorry. I wasn't really following your suggestion. Why is a regex better here?

Reply:
Not a big reason or anything, but just to avoid calling it twice in a row. With regex we can get it done in one go.

Reply:
Ah, got it. I didn't realize it would replace them both. That sounds like a good improvement to me